visualization_scripts/output.sif

```diff
@@ -0,0 +1,413 @@
+EFNA5_HUMAN	N	EPHA7_HUMAN
```

Comment:
We may want to document where this file came from originally and name it to make it more obvious it is example or test data.